sources/web/datalab/polymer/components/datalab-files/datalab-files.ts

```diff
@@ -392,90 +392,87 @@ class FilesElement extends Polymer.Element {
    */
   async _upload() {
```

Comment:
The polymer element contains all of the UI as well as underlying logic. Plus, this logic here is all UI related, it's mostly building the warning message if any and displaying it for large files.

I'm afraid tests will have to come as a separate change.